### PR TITLE
zdiff: add page

### PR DESCRIPTION
**Pull request by gutjuri**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/zdiff.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# zdiff
+
+> Invoke `diff` on gzipped files.
+> More information: <https://manned.org/zdiff>.
+
+- Compare two files, uncompressing them if necessary:
+
+`zdiff {{path/to/file1.txt.gz}} {{path/to/file2.txt.gz}}`
```
Comment (kbdharun) [inline pages/common/zdiff.md:8]:
```suggestion
`zdiff {{path/to/file1.gz}} {{path/to/file2.gz}}`
```
I think `.txt` extension isn't required over here.
[Reference](https://www.geeksforgeeks.org/zdiff-command-in-linux-with-examples/)

File: pages/common/zdiff.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- Invoke `diff` on `path/to/file.txt` and `path/to/file.txt.gz` (which is assumed to exist):
```
Comment (kbdharun) [inline pages/common/zdiff.md:10]:
```suggestion
- Invoke `diff` on `path/to/file.txt` and `path/to/file.txt.gz` (assuming they exist):
```
I am not sure about this command description. Let me ask someone.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, I have some suggestions for this PR.

File: pages/common/zdiff.md (hunk 2 above)
Comment (navarroaxel) [reply pages/common/zdiff.md:10]:
```suggestion
- Compare a file with a gzipped archive with the same name:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**